### ISSUE TITLE
Build error: The command powershell exited with code 9009.

### ISSUE THREAD

**Issue by KulaGGin** [ReClassNET/ReClass.NET#223]
When trying to build in Batch mode in Release configuration projects NativeCore x86 and x64, Reclass.NET x86 and x64 and Reclass.NET_Launcher, I get this error:
> Severity	Code	Description	File	Line	Suppression State
Error		The command "powershell -Command "((Get-Date).ToUniversalTime()).ToString(\"yyyy\/MM\/dd HH:mm:ss\") | Out-File 'D:\Projects\Reversing\ReClass.NET\ReClass.NET\Resources\BuildDate.txt'"
" exited with code 9009.

![devenv_LruxAqJmI8](https://user-images.githubusercontent.com/16989713/166248772-fe2af6fc-a52c-4b86-bf8f-43a0e668a82b.png)

When I remove the pre-build powershell command from the pre-build events, I start to get this error:
> Severity	Code	Description	File	Line	Suppression State
Error		Invalid Resx file. Could not find file 'D:\Projects\Reversing\ReClass.NET\ReClass.NET\Resources\BuildDate.txt'. Line 381, position 5.	D:\Projects\Reversing\ReClass.NET\ReClass.NET\Properties\Resources.resx	381

![devenv_DFNTMGT09R](https://user-images.githubusercontent.com/16989713/166249081-1b572cba-aef3-4c4a-ba60-b3bc53bdc18e.png)

After I manually create empty `BuildDate.txt` file at the specified path, the solution seems to start to build correctly and I can start the program. Don't know if it will function correctly.

If I manually add the `BuildDate.txt` file and restore the powershell command, I still get the `9009` error when trying to build.

Powershell is functional:
![powershell_HDQ0VX9cw8](https://user-images.githubusercontent.com/16989713/166250641-ad26f705-0878-4600-859f-7a2a9699afc8.png)

Windows 10, Visual Studio 2019 and this problem occurs on the last commit of the master branch at the time of writing.

**Comment by KN4CK3R** [comment]
Does `powershell` work in an old `cmd` window?
![grafik](https://user-images.githubusercontent.com/1666336/166250964-98a44c53-2ff9-4add-9dd2-f62b4ef91f78.png)

My search engine tells me that `9009` means `not found`.

**Comment by KulaGGin** [comment]
Thanks. I was missing the path to the folder that has powershell in the `Path` environment variable. I tried launching the powershell's command line, it worked and I tried running powershell script from inside Visual Studio but I haven't tried running powershell from the command line before creating the issue.

After I added the folder to the `Path` environment varialble and restarted Visual Studio, the solution started to build with the powershell script.
